bciers/apps/reporting/src/app/components/changeReview/ChangeReviewForm.tsx

```diff
@@ -7,19 +7,22 @@ import { ReviewChanges } from "@reporting/src/app/components/changeReview/templa
 import { useRouter } from "next/navigation";
 import { ChangeItem } from "@reporting/src/app/components/changeReview/constants/types";
 import ReasonForChangeForm from "@reporting/src/app/components/changeReview/templates/ReasonForChange";
-
 interface ChangeReviewProps {
   versionId: number;
   initialFormData: any;
   navigationInformation: NavigationInformation;
   changes: ChangeItem[];
+  showChanges: boolean;
+  isReportingOnly?: boolean;
 }
 
 export default function ChangeReviewForm({
   versionId,
   initialFormData,
   navigationInformation,
   changes,
+  showChanges = true,
+  isReportingOnly,
```

Comment:
did you mean to set the default value of `true` on the optional `isReportingOnly` param instead of `showChanges` ? (which is required, so the default value will always be overwritten)